tutorials/monitoring/changedetection-1-basics/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Changedetection Basics
+description: Learn the basic features of Changedetection, including monitoring websites and receiving notifications.
+layout: page
+parent: Monitoring
+grand_parent: Tutorials
+---
+
+# How to Monitor Website Changes with Changedetection?
+
+Changedetection is an open-source app that lets you monitor website changes. You can automate the entire process. From simply monitoring website pages that have a change (such as watching prices, restock notification), to deep inspection such as PDF text support, JSON and XML monitoring and extensive text triggers. Here is our step-by-step guide on how you can set up website monitoring.
+
+# How to add and monitor a website with Changedetection?
```

Comment:
Since the whole page is about Changedetection, we don't need to mention it in every title.